examples/calendar/backend/privacy/calendar_js/witness_calculator.js

```diff
@@ -0,0 +1,307 @@
+module.exports = async function builder(code, options) {
+
+    options = options || {};
+
+    let wasmModule;
+    try {
+	wasmModule = await WebAssembly.compile(code);
+    }  catch (err) {
+	console.log(err);
+	console.log("\nTry to run circom --c in order to generate c++ code instead\n");
+	throw new Error(err);
+    }
+
+    let wc;
+
+    
+    const instance = await WebAssembly.instantiate(wasmModule, {
+        runtime: {
+            exceptionHandler : function(code) {
+                let errStr;
+                if (code == 1) {
+                    errStr= "Signal not found. ";
+                } else if (code == 2) {
+                    errStr= "Too many signals set. ";
+                } else if (code == 3) {
+                    errStr= "Signal already set. ";
+		} else if (code == 4) {
+                    errStr= "Assert Failed. ";
+		} else if (code == 5) {
+                    errStr= "Not enough memory. ";
+		} else if (code == 6) {
+                    errStr= "Input signal array access exceeds the size";
+		} else {
+		    errStr= "Unknown error\n";
+                }
+		// get error message from wasm
+		errStr += getMessage();
+                throw new Error(errStr);
+            },
+	    showSharedRWMemory: function() {
+		printSharedRWMemory ();
+            }
+
+        }
+    });
+
+    const sanityCheck =
+        options
+//        options &&
```

Comment:
is this a WIP? If not, please delete

Reply:
This is an auto generated file by circom commands and not manually written.